sycl/test-e2e/Matrix/element_wise_all_ops_half_impl.hpp

```diff
@@ -1,3 +1,11 @@
+//===--------------------- DPC++ joint_matrix------------------------------===//
```

Comment:
please, add file names, see for example here: https://github.com/intel/llvm/blob/4b9819f0b6bf50da03ddc9987918c8d6e09ee757/sycl/test-e2e/Matrix/element_wise_abc_impl.hpp#L1